interview-questions.md

```diff
@@ -7,48 +7,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer:
+  Your answer: It is a snapshot of the object in a class
```

Comment:
Just about. I know your researched answer covers this a bit more though, regarding the scope of the instance variable.